### PR TITLE
finished task 6 and adjusted the order of functions and tests

### PR DESCRIPTION
**Pull request by tchan70**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -68,45 +99,59 @@ describe("GET /api/articles/:article_id", () =>{
...
+        expect(response.body.msg).toBe('article does not exist');
+      })
+  })
+  test('Should send an appropriate status and error message when given an invalid id', () => {
+    return request(app)
+      .get('/api/articles/not-an-article/comments')
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe('Bad request');
+      })
```
Comment (nc-kirsty-butterworth) [inline __tests__/app.test.js:155]: You need another test in here Status 200 : Valid ID but no comments responds with an empty array.

**Review by nc-kirsty-butterworth — COMMENTED** [review]
Nice work so far. Please add the additional test and resubmit 👍
